src/styles/_global.scss

```diff
@@ -10,3 +10,14 @@
     grid-template-columns: repeat(8, 1fr);
   }
 }
+
+.overlay-is-active {
+  height: 100%;
```

Comment:
Created these classes to add to html's classList if the overlay is active or not.